examples/sdk/build_sdk_example_runner.sh

```diff
@@ -17,15 +17,44 @@ readonly EXECUTORCH_ROOT="${SCRIPT_DIR}/../.."
 # Allow overriding the number of build jobs. Default to 9.
 export CMAKE_BUILD_PARALLEL_LEVEL="${CMAKE_BUILD_PARALLEL_LEVEL:-9}"
 
+BUILD_COREML=OFF
+
+usage() {
+  echo "Builds sdk example runner."
+  echo "Options:"
+  echo "  --coreml             Include this flag to enable Core ML backend when building the SDK."
+  exit 0
+}
+
+for arg in "$@"; do
+  case $arg in
+      -h|--help) usage ;;
+      --coreml) BUILD_COREML=ON ;;
+      *)
+  esac
+done
+
 main() {
   cd "${EXECUTORCH_ROOT}"
 
-  rm -rf cmake-out
```

Comment:
Is this intentionally removed?

Reply:
Sorry, not intentional will add it back.